CoreBluetoothMock/CBMPeripheralSpecDelegate.swift

```diff
@@ -251,155 +251,113 @@ public protocol CBMPeripheralSpecDelegate {
 public extension CBMPeripheralSpecDelegate {
 
     func reset() {
-        // Empty default implementation
+        // No-op.
```

Comment:
The intention of "Empty default implementation" was, that these methods are to be overriden by the mock implementation. This kinda suggests "please implement it if you want", while "no-op" more like "this should do nothing". Or am I crazy? :)